src/core_modules/capture-core/components/TemplateSelector/TemplateSelector.component.tsx

```diff
@@ -116,4 +114,4 @@ const TemplateSelectorPlain = (props: Props) => {
     );
 };
 
-export const TemplateSelector = withStyles(getStyles)(TemplateSelectorPlain);
+export const TemplateSelector = withStyles(styles)(TemplateSelectorPlain) as ComponentType<Props>;
```

Comment:
Since we're using the `createStyles` utility function, it does not look like we need `as ComponentType<Props>` anymore. Can you remove that from all files in this PR and make sure it uses the `createStyles` utility function instead.